guides/v2.2/cloud/project/project-conf-files_services.md

```diff
@@ -99,3 +99,17 @@ For example, the current default storage amount per project is 5GB, or 5120MB. Y
 ## Using the services
 
 For services to be available to an application in your project, you must specify [*relationships*]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#relationships) between applications and services in the `.magento.app.yaml` file.
+
+## Service versions available
+
+The following table lists the services used in {{site.data.var.ece}} and their version compatibility with the Magento application.
+
+Service   | Supported Versions
+--------- | -------------------
+`mariadb` | 10.0 to 10.2
```

Comment:
elasticsearch is missed
```
        ServiceInterface::NAME_ELASTICSEARCH => [
            '<2.2.0' => '~1.7.0 || ~2.4.0',
            '>=2.2.0 <2.2.8 || 2.3.0' => '~1.7.0 || ~2.4.0 || ~5.2.0',
            '>=2.2.8 <2.3.0 || >=2.3.1' => '~1.7.0 || ~2.4.0 || ~5.2.0 || ~6.5.0',
        ],
```

Reply:
Ha! Good catch. Let me add that in. I'm not sure what happened...I think when I alphabetized them. Poor Elasticsearch got left out. I'm on it!